src/ESPEasyWiFiEvent.cpp

```diff
@@ -39,7 +39,10 @@ void setUseStaticIP(bool enabled) {
 
 void markGotIP() {
   lastGetIPmoment = millis();
+  // Create the 'got IP event' so mark the wifiStatus to not have the got IP flag set
+  // This also implies the services are not fully initialized.
   wifiStatus     &= ~ESPEASY_WIFI_GOT_IP;
+  wifiStatus     &= ~ESPEASY_WIFI_SERVICES_INITIALIZED;
```

Comment:
use bitRead/bitWrite macros, do you use here inversed logic?

Reply:
That's perhaps indeed _a bit_ more readable :)